0.download_data/README.md

```diff
@@ -0,0 +1,12 @@
+# Download Data
+
+## About the data
```

Comment:
Where do the data come from? Would it be worth providing a mention here to acknowledge those who provided the data? Alternatively, consider adding a [CITATION.cff file](https://citation-file-format.github.io/) to help outline this detail within the `references` section.

Reply:
Added links to the Collaborating companies.